stargate/crates/stargate-lib/src/common.rs

```diff
@@ -50,6 +54,21 @@ impl Opt {
         } else {
             buf.push_str("tracing: disabled");
         }
+        buf.push('\n');
```

Comment:
the `with_capacity` here is surprising, i don't think we need it. i also think refactoring this into a format string would improve readability!